master/internal/api_tensorboard.go

```diff
@@ -286,6 +286,10 @@ func (a *apiServer) LaunchTensorboard(
 	if err != nil {
 		return nil, err
 	}
+
+	if launchReq.Spec.Base.ExtraEnvVars == nil {
```

Comment:
curse go for the zero value of slices being usable but the zero value for maps not, haha.